src/imageManager/ImageManager.js

```diff
@@ -207,6 +213,9 @@ export default class ImageManager extends Component {
         (!data.alt && data.url) && (data.alt = "");
         this.setState(data);
         data.url && this.getImageSize(data, !!data.width);
+        if(data.pieceData){
```

Comment:
on what condition we don't have pieceData here? We should always have it, even from rich text image inserting.
b.t.w. rich text image insert functionality is not passing it currently, but it should. Add it there too